### PR TITLE
Added first functional and integration  tests

### PR DESCRIPTION
**Pull request by marceldobrica**
I have skipped all functional and integration test to bypass github action.... is correct or no?????

### CODE REVIEW

File: src/DataFixtures/AppFixtures.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager): void
+    {
+        // $product = new Product();
+        // $manager->persist($product);
```
Comment (Raresmldvn) [inline src/DataFixtures/AppFixtures.php]: you need to remove this class entirely if you really don't persist anything here

File: tests/Integration/HttpClient/HttpClientSmsUserTest.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+namespace App\Tests\Integration\HttpClient;
+
+use App\HttpClient\HttpClientSmsUser;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\HttpClient\Exception\ClientException;
+
+class HttpClientSmsUserTest extends KernelTestCase
+{
+    private ?HttpClientSmsUser $client;
```
Comment (Raresmldvn) [inline tests/Integration/HttpClient/HttpClientSmsUserTest.php]: why is this nullable? you initialize it in the setup

**Review by Raresmldvn — CHANGES_REQUESTED** [review]
If you have time, maybe you can write some more tests

File: src/DataFixtures/AppFixtures.php (hunk 1 above)
Comment (marceldobrica) [reply src/DataFixtures/AppFixtures.php]: Solved

File: tests/Integration/HttpClient/HttpClientSmsUserTest.php (hunk 2 above)
Comment (marceldobrica) [reply tests/Integration/HttpClient/HttpClientSmsUserTest.php]: Right.

**Pull request merged**